server/model/taggedBailRemoveModel.ts

```diff
@@ -15,11 +15,13 @@ export default class TaggedBailRemoveModel {
   }
 
   public rows() {
+    const caseDetailsHTML = isSentenceRecalled(this.sentenceAndOffence.sentenceCalculationType)
+      ? {
+          html: `${this.getCourtName()} ${getSentenceRecallTagHTML()}<br>${this.getCaseReference()} ${this.getSentenceDate()}`,
+        }
+      : { html: `${this.getCourtName()}<br>${this.getCaseReference()} ${this.getSentenceDate()}` }
```

Comment:
same here

Reply:
I've updated the code as you've suggested above